server/routes/users.js

```diff
@@ -25,20 +25,7 @@ export default (storage, scriptManager) => {
       request: {
         user: req.user
       },
-      payload: {
-        email: req.body.email,
-        username: req.body.username,
-        connection: req.body.connection,
-        memberships: req.body.memberships,
-        password: req.body.password
-      },
-      defaultPayload: {
```

Comment:
Why is this removed?

Reply:
It was removed because with custom fields there are more fields that need to be able to be set and it is configured in the settings hook.  We could potentially set this only based on whether or not the userFields array is blank or not.